### ISSUE TITLE
[GENERIC] What do we do about non-protocol projects?

### ISSUE THREAD

**Issue by andrewhong5297** [andrewhong5297/Crypto-Grants-Analysis#2]
There will be many non-protocol projects (developer tooling, communities, etc). Thoughts on how we should measure them? maybe shouldn't be included in analysis (can still include in uploads).

**Comment by chuxinh** [comment]
Agreed that they should still be included in the uploads. I would suggest having some pre-defined label of `verticals` that we can classify various projects into, so that it's easier to view if that's a developer tool, defi etc, and understand the funding allocation

Even knowing if that's protocol or non protocol would be nice too
